plugins/ScheduledReports/vue/src/ListReports/ListReports.vue

```diff
@@ -0,0 +1,246 @@
+        <th>{{ translate('General_Edit') }}</th>
+        <th>{{ translate('General_Delete') }}</th>
+      </tr>
+      </thead>
+      <tbody>
+        <tr v-if="userLogin === 'anonymous'">
+          <td colspan="7">
+            <br />
+            {{ translate('ScheduledReports_MustBeLoggedIn') }}
+            <br />&rsaquo; <a :href="`index.php?module=${loginModule}`">
+              {{ translate('Login_LogIn') }}
+            </a>
+            <br /><br />
+          </td>
+        </tr>
+        <tr v-else-if="!reports?.length">
+          <td colspan="7">
+            <br />
+            {{ translate('ScheduledReports_ThereIsNoReportToManage', siteName) }}.
+            <br /><br />
+          </td>
+        </tr>
+        <tr v-for="report in reports" :key="report.idreport">
+          <td class="first">
+            {{ report.description }}
+            <div
+              class="entityInlineHelp"
+              style="font-size: 9pt;"
+              v-if="segmentEditorActivated && report.idsegment"
+            >
+              <span v-if="savedSegmentsById[report.idsegment]">
+                {{ savedSegmentsById[report.idsegment] }}
+              </span>
+              <span v-else>
+                {{ translate('ScheduledReports_SegmentDeleted') }}
+              </span>
+            </div>
+          </td>
+          <td>{{ periods[report.period] }}
+            <!-- Last sent on {{ report.ts_last_sent }} -->
+          </td>
+          <td>
+            <span v-if="report.format">{{ report.format.toUpperCase() }}</span>
+          </td>
+          <td>
+            <span v-if="report.recipients.length === 0">
+              {{ translate('ScheduledReports_NoRecipients') }}
+            </span>
+            <span v-for="(recipient, index) in report.recipients" :key="index">
+              {{ recipient }}
+              <br />
+            </span>
+
+            <a
+              v-if="report.recipients.length !== 0"
+              href="#"
+              name="linkSendNow"
+              class="link_but withIcon"
+              style="margin-top:3px;"
+              @click.prevent="$emit('sendnow', report.idreport)"
+            >
+              <img
+                border="0"
+                :src="reportTypes[report.type]"
+              />
+              {{ translate('ScheduledReports_SendReportNow') }}
+            </a>
+          </td>
+          <td>
+            <form
+              method="POST"
+              target="_blank"
+              :id="`downloadReportForm_${report.idreport}`"
+              :action="linkTo({
+                module: 'API',
+                segment: null,
+                method: 'ScheduledReports.generateReport',
+                idReport: report.idreport,
+                outputType: downloadOutputType,
+                language: language,
+                format: ['html', 'csv', 'tsv'].indexOf(report.format) !== -1
+                  ? report.format : undefined,
```

Comment:
We maybe could use `original` here as default. Currently downloading a SMS report won't work, as the API request doesn't contain a format and uses `XML` as default.